gravitee-gateway-standalone/gravitee-gateway-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -386,3 +386,8 @@ ds:
 #  hazelcast:
 #    config:
 #      path: ${gravitee.home}/config/hazelcast.xml
+
+# Encrypt API properties using this secret
+apiProperties:
```

Comment:
`apiProperties` seems weird. Do you think it could be changed with:
```
api:
  properties:
    encryption:
      secret: vvLJ4Q8Khvv9tm2tIPdkGEdmgKUruAL6
```